README.md

```diff
@@ -522,6 +522,24 @@ Advanced users may call the Python client via `async` and `await` syntax. The
 [stream](src/python/examples/simple_grpc_aio_sequence_stream_infer_client.py)
 examples demonstrate how to infer with AsyncIO.
 
+
+### Request Cancellation
+
+Starting from r23.10, triton python gRPC client can issue cancellation
+to inflight requests. This can be done by calling `cancel()` on the
+CallContext object returned by `async_infer()` API.
+For streaming requests, `cancel_requests=True` can be sent to
+`stop_stream()` API to terminate all the inflight requests
+sent via this stream. See more details about these APIs in
+[grpc/\_client.py](src/python/library/tritonclient/grpc/_client.py).
```

Comment:
I think it would be clearer to just show `client.stop_stream(cancel_requests=True)` rather than saying "the arg can be sent to the API", for a reader.

Reply:
Or a small code block:
```python
  client.start_stream()
  for _ in range(10):
    client.async_stream_infer(...)

  # Attempt to cancel requests on stream closure rather than blocking until requests complete
  client.stop_stream(cancel_requests=True)
```